hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/lockconfig"
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/constants"
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/utils"
+)
+
+var packageRepository, version, subVersion string
+var all bool
+
+// packageBundleGenerateCmd is for generating package bundle
+var packageBundleGenerateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate package bundle",
+	RunE:  runPackageBundleGenerate,
+}
+
+func init() {
+	packageBundleCmd.AddCommand(packageBundleGenerateCmd)
+	packageBundleGenerateCmd.Flags().StringVarP(&packageRepository, "repository", "", "", "Package repository of the package bundle being created")
+	packageBundleGenerateCmd.Flags().StringVarP(&version, "version", "", "", "Package bundle version")
+	packageBundleGenerateCmd.Flags().StringVarP(&subVersion, "sub-version", "", "", "Package bundle subversion")
+	packageBundleGenerateCmd.Flags().BoolVarP(&all, "all", "", false, "Generate all package bundles in a repository")
+	packageBundleGenerateCmd.MarkFlagRequired("repository") //nolint
```

Comment:
Say which linter is being skipped with the nolint (`//nolint: foo`). Another option is to change these to:

```
_ = packageBundleGenerateCmd.MarkFlagRequired("repository")
```

That makes it explicit that you don't care about the returned value.

Reply:
Yeah, either one works, so we just need to do one or the other. The change to `//nolint: errcheck` works great. Thanks!